### PR TITLE
executor: skip unstable test case TestMergeJoinInDisk and TestAnalyzeIndexExtractTopN.

### PR DESCRIPTION
**Pull request by bb7133**
### What problem does this PR solve?

Issue Number: related to https://github.com/pingcap/tidb/issues/25157

Problem Summary:

What's Changed: the unstable test cases are skipped temporarily.

How it Works:

### Check List 

Tests 

- NA

### Release note 

- No release note

### CODE REVIEW

**Review by hi-rustin — APPROVED** [review]

**Review by youjiali1995 — APPROVED** [review]

File: executor/merge_join_test.go [hunk 1]
```diff
@@ -279,6 +279,8 @@ func (s *testSerialSuite1) TestShuffleMergeJoinInDisk(c *C) {
 	c.Assert(tk.Se.GetSessionVars().StmtCtx.DiskTracker.MaxConsumed(), Greater, int64(0))
 }
 func (s *testSerialSuite1) TestMergeJoinInDisk(c *C) {
+	c.Skip("unstable, skip race test")
```
Comment (tiancaiamao) [inline executor/merge_join_test.go:282]: Race test or all the tests?

Comment (bb7133) [reply executor/merge_join_test.go]: addressed, thanks.

**Review by tiancaiamao — APPROVED** [review]

**Review by youjiali1995 — APPROVED** [review]

**Review by hi-rustin — APPROVED** [review]

**Pull request merged**